crnk-core/src/main/java/io/crnk/core/boot/CrnkBoot.java

```diff
@@ -161,6 +163,26 @@ public void boot() {
 		bootDiscovery();
 	}
 
+	private void setupQuerySpecDeserializer() {
+		if (this.querySpecDeserializer == null) {
```

Comment:
maybe two ifs would be better, one to setup the default one. And the other to set all the properties regardless whether the default instance or a custom one is in use. If the non-custom one is in use and people do not want that. The application can simply ignore the properties.

Reply:
Could you specify please? The idea that no need to configure Crnk to use default instance, it will be used implicitely if no custom bean found